{{ cookiecutter.hosting_github_repo_name }}/README.md

```diff
@@ -0,0 +1,141 @@
+        _Do not save this token as a file in this repository's directory._
+11. Create the GitHub repository for this deployment to live in.
+    [Create a new Repository](https://github.com/new)
+    -   Set the repo name to: **{{ cookiecutter.hosting_github_repo_name }}**
+    -   Set the repo owner to: **{{ cookiecutter.hosting_github_username_or_org }}**
+    -   Set the repo visibility to: "Public"
+    -   Do not initialize with any of the extra options
+    -   Click "Create repository".
+12. Initialize and push the local repository to GitHub.
+    -   In a terminal, while in this repository's directory, run:
+        ```bash
+        git init
+        git add -A
+        git commit -m "Initial commit"
+        git branch -M main
+        git remote add origin https://github.com/{{ cookiecutter.hosting_github_username_or_org }}/{{ cookiecutter.hosting_github_repo_name }}.git
+        git push -u origin main
+        ```
+    -   Refresh your repository's dashboard to ensure that all files were pushed.
+13. Configure [repository settings](https://github.com/{{ cookiecutter.hosting_github_username_or_org }}/{{ cookiecutter.hosting_github_repo_name }}/settings).
+    -   In the "Options" tab, configure "GitHub Pages"
+        -   Select "gh-pages" from the "Source" dropdown.
+        -   Click "Save".
+    -   In the "Secrets" tab, configure two "Actions secrets"
+        -   Click "New repository secret".
+        -   Set "Name" to: **GOOGLE_CREDENTIALS**
+        -   Open up the file downloaded from Google Cloud (the Service Account JSON),
+            copy all text in the file, paste the text into the "Value" field.
+        -   Click "Add secret".
+        -   Click "New repository secret".
+        -   Set "Name" to: **PULUMI_ACCESS_TOKEN**
+        -   Open up the file or browser tab where you saved your Pulumi token,
+            copy the token, past the token into the "Value" field.
+        -   Click "Add secret".
+14. Navigate to [GitHub Actions History](https://github.com/JacksonMaxfield/cdp-jackson-dev-stack/actions).
+
+    -   Click on the "workflow run" tagged with "Initial commit" and "Infrastructure".
+    -   Click the "Re-run jobs" dropdown.
+    -   Click "Re-run all jobs".
+
+**If all steps complete successful your web application will be viewable at:
+https://{{ cookiecutter.hosting_github_username_or_org }}.github.io/{{ cookiecutter.hosting_github_repo_name }}**
+
+Once all of these steps are done, feel free to delete this section of the README.
+
+## Data Gathering Setup
+
+Once your repository, infrastructure, and web application have been set up,
+you will need to write an event data gathering function.
+
+Navigate and follow the instructions in the the file:
+`python/cdp_{{ cookiecutter.python_municipality_slug }}_backend/scraper.py`.
+
+As soon as you push your updates to your event gather function to your GitHub
+repository, everything will be tested and configured for the next pipeline run.
```

Comment:
I think we should mention the `get_events` function here. Something like `updates to your event gather function, get_events`.

I know the event ingestion models documentation is already linked, but I think it wouldn't hurt to link it here again.

Reply:
Yeah I should have added that there will be a PR later to fix and improve the documentation specific to `get_events`. Wanted to split those up. But at the very least will add this.